restservice/src/main/java/org/itevents/controller/UserRestController.java

```diff
@@ -43,8 +38,7 @@
     @ResponseStatus(value = HttpStatus.OK)
     public TokenWrapper login(@ModelAttribute("username") String username,
                               @ModelAttribute("password") String password) {
-        String token = tokenService.createToken(username, password);
-        return new TokenWrapper(token);
+        return tokenService.createTokenWrapper(username, password);
```

Comment:
i think this should be refactored. I think its better to make this a private method of tokenService.login(String username, String password)